src/hardware/mixer.cpp

```diff
@@ -347,7 +347,10 @@ static void set_global_crossfeed(const MixerChannelPtr& channel)
 {
 	assert(channel);
 
-	if (!mixer.do_crossfeed || !channel->HasFeature(ChannelFeature::Stereo)) {
+	const auto apply_crossfeed = (channel->GetName() == ChannelName::Opl) ||
+	                             (channel->GetName() == ChannelName::Cms);
+
+	if (!mixer.do_crossfeed || !apply_crossfeed) {
```

Comment:
I think you may need to leave the Stereo check in. OPL2 is mono:

https://github.com/dosbox-staging/dosbox-staging/blob/f65827490fac96577addb4afeb5734ad4dc9fdbe/src/hardware/opl.cpp#L841-L845

Reply:
Ah yeah, forgot about that.

Reply:
Fixed. Also updated the top description with a test case.